user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/BatchRegistrationDialog.java

```diff
@@ -139,14 +67,15 @@ public BatchRegistrationDialog(String experimentName,
 
     Div prefillSection = new Div();
     Button prefillSpreadsheet = new Button();
-    prefillSpreadsheet.setText("Prefill Spreadsheet");
-    prefillSpreadsheet.setAriaLabel("Prefill complete sample batch");
+    prefillSpreadsheet.setText("Clear and Prefill");
+    prefillSpreadsheet.setAriaLabel("Clear and Prefill");
     prefillSpreadsheet.addClickListener(this::onPrefillClicked);
     prefillSpreadsheet.addClassName("prefill-batch");
 
     Span prefillText = new Span(
-        "Do you want to register a batch containing all biological replicates? You can prefill information already know to the system."
-    );
+        "Do you want to register a batch containing all biological replicates? "
+            + "You can prefill information already know to the system. "
+            + "Please note: This will erase any information entered into the spreadsheet.");
```

Comment:
This change is not adherent to the prototype. 
Was this discussed with Shraddha?

Reply:
No but as she is not here, we can talk about it when she is back. As the styling needs to be adjusted anyhow, I would like to leave this as is.